pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -10312,4 +10312,16 @@ Recognized isotopes include: {2}</value>
   <data name="SkylineVersion_V4_2_Skyline_4_2" xml:space="preserve">
     <value>Skyline 4.2</value>
   </data>
-</root>
+  <data name="BuildPeptideSearchLibraryControl_BuildPeptideSearchLibrary_Embedded" xml:space="preserve">
+    <value>Embedded</value>
+  </data>
+  <data name="BiblioSpecLiteBuilder_Could_not_find_an_external_spectrum_file_matching__0__in_the_same_directory_as_the_MaxQuant_input_file__1__" xml:space="preserve">
+    <value>Could not find an external spectrum file matching '{0}' in the same directory as the MaxQuant input file '{1}', or in its parent or grandparent directory.
+
+If you do not have the original file, you may build the library with embedded spectra from the input file. However, fragment ions in MaxQuant embedded spectra are charge state deconvoluted, and will contain only singly charged fragment ions which may not be representative of intensities measured by a mass spectrometer.
+

```

Comment:
Split this message into two pieces with the first part used in both the UI and the CLI. The second part for CLI:

Use the argument --import-search-prefer-embedded-spectra to force the library build to use embedded spectra.

Reply:
Like this?

> Opening file...
> File blank.sky opened.
> Creating spectral library from files:
> yeast-wiff-msms.txt
> Could not find an external spectrum file matching 'wine yeast sampleA_2' in the same directory as the MaxQuant input file 'C:\pwiz.git\pwiz\pwiz_tools\Skyline\bin\x64\Release\CommandLineImportTest\yeast-wiff-msms.txt', or in its parent or grandparent directory.
> 
> If you do not have the original file, you may build the library with embedded spectra from the input file. However, fragment ions in MaxQuant embedded spectra are charge state deconvoluted, and will contain only singly charged fragment ions which may not be representative of intensities measured by a mass spectrometer.
> 
> Use the argument --import-search-prefer-embedded-spectra to force the library build to use embedded spectra, or place the original spectrum files next to the input files (with a supported file extension: mz5, mzML, raw, wiff, d, lcd, mzXML, cms2, ms2, or mgf) and rerun.
>

Reply:
Yeah. That looks about right. If you push your changes I will have a look.